lib/max/Delivery/adSelect.php

```diff
@@ -238,8 +238,8 @@ function MAX_adSelect($what, $campaignid = '', $target = '', $source = '', $with
             'bannerid'      => $row['bannerid'],
             'contenttype'   => $row['contenttype'],
             'alt'           => $row['alt'],
-            'width'         => $row['width'],
-            'height'        => $row['height'],
+            'width'         => (!isset($row['width']) || is_null($row['width'])) ? 0 : $row['width'],
+            'height'        => (!isset($row['height']) || is_null($row['height'])) ? 0 : $row['height'],
```

Comment:
Thanks for the PR. Simply `empty($row['height']) ? 0 : $row['height']` would be shorter here